drivers/Web/httpResponse.py

```diff
@@ -21,8 +20,15 @@ def responseAsBytes(response):
     headers["Content-length"] = len(body)
     status = response.getStatus()
     version = "1.1"
-    mappingStatus = {200: "OK", 303: "See Other", 404: "Not Found", 405: "Method Not Allowed"}
-    responseToBytes = f"HTTP/{version} {status} {mappingStatus.get(status, '')}\r\n"
+    mappingStatus = {
+        200: "OK",
+        303: "See Other",
+        404: "Not Found",
+        405: "Method Not Allowed"
+    }
+    responseToBytes = f"HTTP/{version}" \
+                      f" {status}" \
+                      f" {mappingStatus.get(status, '')}\r\n"
```

Comment:
Maybe you can create a variable like completeStatus = {status} {statusMsg} where  statusMessage get the mappingStatus?